deployment/postgres-init-db/sql/tables/sequencing/parcel.sql

```diff
@@ -0,0 +1,46 @@
+create table sequencing.parcel (
+  id integer generated always as identity,
+
+  name text not null,
+
+  command_dictionary_id integer not null,
+  channel_dictionary_id integer default null,
+  sequence_adaptation_id integer default null,
+
+  created_at timestamptz not null default now(),
+  updated_at timestamptz not null default now(),
+
+  owner text,
+
+  constraint parcel_synthetic_key
+    primary key (id),
+
+  foreign key (channel_dictionary_id)
+    references sequencing.channel_dictionary (id),
+  foreign key (command_dictionary_id)
+    references sequencing.command_dictionary (id),
+  foreign key (sequence_adaptation_id)
+    references sequencing.sequence_adaptation (id),
```

Comment:
All of these are missing `on update` and `on delete`

`command_dictionary` should have a `cascade` on delete, the other two should have `set null`